book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -249,163 +542,29 @@ Only if this section is defined a sensitivity analysis is done. This section wil
 </sensitivity.analysis>
 ```
 
-* **quantiles** : [optional] Quantiles of parameter distributions at which the model should be evaluated when running sensitivity analysis. Values greater than 0 and less than 1 can be used.  
-* **sigma** : [optional] Any real number can be used to indicate the quantiles to be used in units of normal probability.  
-* **quantile** : [optional] Which quantile should be used.  
-* **start.date** : [required?] start date of the sensitivity analysis (in YYYY/MM/DD format) 
-* **end.date** : [required?] end date of the sensitivity analysis (in YYYY/MM/DD format)
+* `quantiles` : [optional] Quantiles of parameter distributions at which the model should be evaluated when running sensitivity analysis. Values greater than 0 and less than 1 can be used.
```

Comment:
I'd rephrase `Values greater than 0 and less than 1 can be used.` to just be `Values must be between 0 and 1`

Reply:
Addressed in 68e5b476d.